stdlib/ballerina-http/src/main/ballerina/http/http_circuit_breaker.bal

```diff
@@ -341,8 +345,9 @@ public function CircuitBreakerClient::patch (string path, Request request) retur
    HttpClient httpClient = self.httpClient;
    CircuitBreakerInferredConfig cbic = self.circuitBreakerInferredConfig;
    self.currentCircuitState = updateCircuitState(self.circuitHealth, self.currentCircuitState, cbic);
-   Response response;
-   HttpConnectorError httpConnectorError;
+   Response response = new;
+    //TODO remove this workaround once record init is introduced
+   HttpConnectorError httpConnectorError = {statusCode:501};
```

Comment:
We shouldn't be creating this here. You can declare it as; HttpConnectorError? httpConnectorError, because this error will be returned only sometimes. So create it when you need to do so.